lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -50,7 +53,7 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, M
         return resolve(latestVersion);
       });
     });
-  };
+  }, _.constant('latest-version'));
```

Comment:
Looks great! Just one last minor thing: can we add a comment above this line clarifying that the `latest-version` string is completely arbitrary and has no relation with the actual function name? For a second, I thought that this id meant something special.